app/js/containers/Artifacts.jsx

```diff
@@ -1,18 +1,30 @@
 import React from 'react';
 import { connect } from 'react-redux';
 import Artifact from '../components/Artifact';
+import { deleteArtifact } from '../actions';
+import Error from '../components/Error';
 
 const mapStateToProps = (state) => ({
     artifacts: state.artifacts
 });
 
 const mapDispatchToProps = (dispatch) => ({
-    onArtifactClick: (id) => {
-        dispatch({ type: 'DELETE_ARTIFACT', id });
-    }
+    boundDeleteArtifact: (id) => dispatch(deleteArtifact(id))
```

Comment:
This isn't really bound to anything (other than dispatch, which is a given). That being said I don't have a good recommendation. I have recently experienced the pain of distinguishing these from "normal" props and do think a naming convention of some kind would be nice.